docs/source/data-model.rst

```diff
@@ -20,31 +32,39 @@ JSON) in any language. Each document class has a defined, but flexible, schema.
 * Event Descriptor --- Metadata about a series of Events. Envision
   richly-detail column headings in a table, encompassing physical units,
   hardware configuration information, etc.
+* Resource --- A pointer to an external file (or resource in general).
+* Datum --- A pointer to a specific slice of data within a Resource.
 * Run Stop Document ---  Everything that we can only know at the very end, such
   as the time it ended and the exit status (succeeded, aborted, failed due to
   error).
 
+Every document has a ``time`` (its creation time) and a separate ``uid`` to
+identify it. The Event documents also have a ``descriptor`` field linking them
+to the Event Descriptor with their metadata. And the Event Descriptor and
+Run Stop documents have a ``run_start`` field linking them to their Run
+Start. Thus, all the documents in a run are linked back to the Run Start.
+
+Event documents may contain the literal values or *pointers* to values that
+are stored in some external file or networked resource, yet to be loaded. The
+Resource and Datum document types manage references to externally-stored data.
+
+Example Runs
+============
+
 .. image:: _static/document-generation-timeline.svg
    :width: 100%
    :align: center
 
-Event documents may contain the literal values or as *pointers* to values that
-are stored in some external file or networked resource, yet to be loaded. Two
-additional document types manage references to externally-stored data.
-
-* Resource --- A pointer to an external file (or resource in general).
-* Datum --- A pointer to a specific slice of data within a Resource.
-
 Finally, Event and Datum can be represented in "paged" form, where multiple
 rows are contained in one structure for efficient transport and vectorized
 computation. The representations contain equivalent information: an EventPage
 can always be losslessly transformed into an Event and vice versa.
 
-The scope of a "run" is left to the user. It is quite generic: a set of
-Documents generated by following a given sequence of instructions. This might
-encompass a single short measurement (say, "a scan") or a multi-step procedure.
-It can be whatever scope is convenient for accessing a unit of data for later
-analysis.
+The scope of a "run" is left to the beamline or individual scientist. It is
```

Comment:
We have users in sciences outside the synchrotron world, and we go out of our way to avoid creating the impression that bluesky is for synchrotrons only.

```suggestion
The scope of a "run" is left to the research group or individual scientist. It is
```

Reply:
> ... is left to the instrument team.

Reply:
*instrument team* has been easier to use and includes anyone who has in interest in this subject - also not specific to "beam line", could be a lab

Reply:
Perfect, those are the words I was looking for.

Reply:
check